samples/snippets/src/main/java/pubsublite/SubscriberExample.java

```diff
@@ -41,65 +40,62 @@
 
   public static void main(String... args) throws Exception {
     // TODO(developer): Replace these variables before running the sample.
-    String CLOUD_REGION = "Your Cloud Region";
-    char ZONE_ID = 'b';
-    String SUBSCRIPTION_NAME = "Your Subscription Name";
-    long PROJECT_NUMBER = Long.parseLong("123456789");
+    String cloudRegion = "your-cloud-region";
+    char zoneId = 'b';
+    // Choose an existing subscription for the subscribe example to work.
+    String subscriptionId = "your-subscription-id";
+    long projectNumber = Long.parseLong("123456789");
     // List of partitions to subscribe to. It can be all the partitions in a topic or
     // a subset of them. A topic of N partitions has partition numbers [0~N-1].
-    List<Integer> PARTITION_NOS = ImmutableList.of(0);
+    List<Integer> partitionNumbers = ImmutableList.of(0);
 
-    subscriberExample(CLOUD_REGION, ZONE_ID, PROJECT_NUMBER, SUBSCRIPTION_NAME, PARTITION_NOS);
+    subscriberExample(cloudRegion, zoneId, projectNumber, subscriptionId, partitionNumbers);
   }
 
   public static void subscriberExample(
-      String CLOUD_REGION,
-      char ZONE_ID,
-      long PROJECT_NUMBER,
-      String SUBSCRIPTION_NAME,
-      List<Integer> PARTITION_NOS)
+      String cloudRegion,
+      char zoneId,
+      long projectNumber,
+      String subscriptionId,
+      List<Integer> partitionNumbers)
       throws Exception {
 
-    CloudRegion cloudRegion = CloudRegion.of(CLOUD_REGION);
-    CloudZone zone = CloudZone.of(cloudRegion, ZONE_ID);
-    ProjectNumber projectNum = ProjectNumber.of(PROJECT_NUMBER);
-    SubscriptionName subscriptionName = SubscriptionName.of(SUBSCRIPTION_NAME);
-
     SubscriptionPath subscriptionPath =
         SubscriptionPaths.newBuilder()
-            .setZone(zone)
-            .setProjectNumber(projectNum)
-            .setSubscriptionName(subscriptionName)
+            .setZone(CloudZone.of(CloudRegion.of(cloudRegion), zoneId))
+            .setProjectNumber(ProjectNumber.of(projectNumber))
+            .setSubscriptionName(SubscriptionName.of(subscriptionId))
             .build();
 
     FlowControlSettings flowControlSettings =
         FlowControlSettings.builder()
-            // Set outstanding bytes to 10 MiB per partition.
+            // 10 MiB. Must be >0. It controls the maximum size of messages the subscriber
+            // receives before pausing the message stream.
```

Comment:
I think there's a warning somewhere in the docs about setting this lower than 1MiB (max message size) not allowing forward progress (i.e. this is a hard limit respected by the server). You might want to duplicate that here?

Reply:
Where is this warning? I couldn't find anything in the library code. Also, when the code is run, I also don't see this warning.

Reply:
https://screenshot.googleplex.com/GQ7kyuiv5Hz

Reply:
Done. We should update Cloud site to indicate max message size is 1MiB.